<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimerW</class>
 <widget class="QDialog" name="TimerW">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>298</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhNone</set>
  </property>
  <widget class="QListWidget" name="list_of_timers">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>16</y>
     <width>401</width>
     <height>152</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QPushButton" name="start_stop_timer">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>174</y>
     <width>401</width>
     <height>24</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>240</red>
         <green>128</green>
         <blue>185</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>240</red>
         <green>128</green>
         <blue>185</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>240</red>
         <green>128</green>
         <blue>185</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <stylestrategy>PreferDefault</stylestrategy>
    </font>
   </property>
   <property name="text">
    <string>стоп</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_timer">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>234</y>
     <width>401</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>видалити</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pause_continue_timer">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>264</y>
     <width>401</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>стоп</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_new_timer">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>204</y>
     <width>401</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>додати</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
